components/financial_assistance/spec/domain/financial_assistance/operations/applications/transformers/application_to/cv3_application_spec.rb

```diff
@@ -1182,7 +1182,7 @@
     end
 
     it 'should be able to successfully return mitc_income with amount' do
-      expect(@mitc_income[:amount]).to eq(60_000.00)
+      expect(@mitc_income[:amount]).to eq(30_000)
```

Comment:
@jayreddy519 why are we changing this to 30,000?

Reply:
net_self_employment is moved to other income, 
- the value for @mitc_income[:amount] will be reduced to 30_000.
- the value for @mitc_income[:other_income] will be increased to 30_000.